### ISSUE TITLE
Hyperion not controlling LED's runtime_error

### ISSUE THREAD

**Issue by jfarago** [tvdzwan/hyperion#169]
Any help would be greatly appreciated. Let me know if any more information is needed. 

Linux 3.12.21
Raspbmc 13.2

sudo hyperiond /etc/hyperion.config.json
Application build time: Aug 24 2014 20:57:16
QCoreApplication initialised
Selected configuration file: /etc/hyperion.config.json
ColorTransform 'default' => [0; 110]
Device configuration: 
{
	"colorOrder" : "rgb",
	"name" : "MyPi",
	"output" : "/dev/spidev0.0",
	"rate" : 250000,
	"type" : "ws2801"
}

Black border threshold set to 0.01 (3)
Not creating any smoothing
Effect loaded: Knight rider
Effect loaded: Blue mood blobs
Effect loaded: Cold mood blobs
Effect loaded: Full color mood blobs
Effect loaded: Green mood blobs
Effect loaded: Red mood blobs
Effect loaded: Warm mood blobs
Effect loaded: Rainbow mood
Effect loaded: Rainbow swirl fast
Effect loaded: Rainbow swirl
Effect loaded: Snake
Effect loaded: Strobe blue
Effect loaded: Strobe Raspbmc
Effect loaded: Strobe white
Initializing Python interpreter
Hyperion created and initialised
run effect Rainbow swirl fast on channel 0
Boot sequence(Rainbow swirl fast) created and started
XBMC video checker created and started
Display opened with resolution: 1920x1080
Frame grabber created and started
terminate called after throwing an instance of 'std::runtime_error'
  what():  Json server could not bind to port

**Comment by nuovocampo** [comment]
I am having a similar problem. However, it seems that the port is not already used by another instance of Hyperion or sth else. I have changed the port multiple times but always got the same error. I just did a fresh install of the latest Raspbian (February 16th). Before I had an install from beginning of January running an no issues, so I am suspecting I might have setup sth wrongly (very new to Linux ;-) ).
Thankful for any tip what I'm overseeing here :)

I can control Hyperion via remote commands through SSH but can't connect via the app.


Linux raspberrypi 3.18.7+ #755 PREEMPT Thu Feb 12 17:14:31 GMT 2015 armv6l GNU/Linux

sudo hyperiond /etc/hyperion.config.json

Application build time: Jan 11 2015 16:59:41
QCoreApplication initialised
Selected configuration file: /etc/hyperion.config.json
ColorTransform 'default' => [0; 133]
Device configuration:
{
	"colorOrder" : "rgb",
	"name" : "MyPi",
	"type" : "ws2812b"
}

Page map: 1 pages
WS2812b init finished
Black border threshold set to 0.01 (3)
Creating linear smoothing
Created linear-smoothing(interval_ms=40;settlingTime_ms=100;updateDelay=0
Effect loaded: Knight rider
Effect loaded: Blue mood blobs
Effect loaded: Cold mood blobs
Effect loaded: Full color mood blobs
Effect loaded: Green mood blobs
Effect loaded: Red mood blobs
Effect loaded: Warm mood blobs
Effect loaded: Rainbow mood
Effect loaded: Rainbow swirl fast
Effect loaded: Rainbow swirl
Effect loaded: Snake
Effect loaded: Strobe blue
Effect loaded: Strobe Raspbmc
Effect loaded: Strobe white
Initializing Python interpreter
Hyperion created and initialised
run effect Rainbow swirl fast on channel 0
Boot sequence(Rainbow swirl fast) created and started
XBMC video checker created and started
Display opened with resolution: 720x480
Frame grabber created and started
terminate called after throwing an instance of 'std::runtime_error'
  what():  Json server could not bind to port


Only one hyperiond running under top:

  PID USER      PR  NI  VIRT  RES  SHR S  %CPU %MEM    TIME+  COMMAND
 2724 root      20   0 92992  18m  15m S   3.6  4.2   2:03.56 hyperiond
 2780 pi        20   0  4676 2516 2136 R   1.3  0.6   0:00.21 top
   30 root       1 -19     0    0    0 S   0.7  0.0   0:14.31 VCHIQ-0
 2535 pi        20   0 96356  18m  16m S   0.7  4.3   0:09.97 lxpanel
   36 root      20   0     0    0    0 S   0.3  0.0   0:00.39 kworker/u2:1
 1724 root      20   0  1756 1224 1140 S   0.3  0.3   0:02.61 ifplugd
    1 root      20   0  2152 1264 1160 S   0.0  0.3   0:01.88 init
    2 root      20   0     0    0    0 S   0.0  0.0   0:00.00 kthreadd
    3 root      20   0     0    0    0 S   0.0  0.0   0:00.36 ksoftirqd/0
    5 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 kworker/0:0H
    7 root      20   0     0    0    0 S   0.0  0.0   0:01.67 rcu_preempt
    8 root      20   0     0    0    0 S   0.0  0.0   0:00.00 rcu_sched
    9 root      20   0     0    0    0 S   0.0  0.0   0:00.00 rcu_bh
   10 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 khelper
   11 root      20   0     0    0    0 S   0.0  0.0   0:00.01 kdevtmpfs
   12 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 netns
   13 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 perf
   14 root      20   0     0    0    0 S   0.0  0.0   0:00.00 khungtaskd
   15 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 writeback
   16 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 crypto
   17 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 bioset
   18 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 kblockd
   20 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 rpciod
   21 root      20   0     0    0    0 S   0.0  0.0   0:00.00 kswapd0
   22 root      20   0     0    0    0 S   0.0  0.0   0:00.00 fsnotify_mark
   23 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 nfsiod
   29 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 kthrotld
   31 root       1 -19     0    0    0 S   0.0  0.0   0:00.17 VCHIQr-0
   32 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 VCHIQs-0
   33 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 iscsi_eh
   34 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 dwc_otg
   35 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 DWC Notificatio
   37 root      20   0     0    0    0 S   0.0  0.0   0:00.00 VCHIQka-0
   38 root      10 -10     0    0    0 S   0.0  0.0   0:00.00 SMIO
   39 root       0 -20     0    0    0 S   0.0  0.0   0:00.00 deferwq
